status-monitor.lic

```diff
@@ -154,6 +154,31 @@ class GameFilter
   end
 end
 
+require 'slack-ruby-client'
```

Comment:
Could you move the slackbot into a separate file? we could then just custom_require it if the slack_username setting exists that way people wanted to use status monitor don't need the gem. So all we'd have here is
```

if get_settings['slack_username']
  custom_require.call('somefile')
  bot = SlackBot.new
end
```